build/SharedFxInstaller.targets

```diff
@@ -222,26 +222,6 @@
   </Target>
 
   <Target Name="GenerateDebs" DependsOnTargets="_EnsureInstallerPrerequisites">
-    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="Version=$(Version);Image=debian.8" />
     <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_GenerateDebOnPlatform" Properties="Version=$(Version);Image=ubuntu.14.04" />
```

Comment:
Curious. Why was ubuntu.14.04 kept but other variants were removed?

Reply:
Mainly to keep in line with what I saw in the core-setup PR https://github.com/dotnet/core-setup/pull/3487, specifically: https://github.com/rakeshsinghranchi/core-setup/blob/2bf97ce26e7c1883707a36a8106aff96cbc07cf1/src/pkg/packaging/deb/package.targets#L49-L52. We now only need to build the .deb installers once and it doesn't matter which platform we build it on. I'm checking with the core-setup team offline right now to see why they chose ubuntu 14.04 but I suspect they just picked it arbitrarily.

Reply:
Yup checked with @rakeshsinghranchi that ubuntu 14.04 was picked arbitrarily.

Reply:
Does this have any effect on the deb produced?

Reply:
@wli3 and @leecow

Reply:
From my testing on debian.8 ad ubuntu.14.04 there didn't seem to be any effect on the produced debs.

Reply:
I don't know enough about the build system to answer. @rakeshsinghranchi will need to speak to how or if the targets impact downstream deb creation (dependency selection, etc).

Reply:
>Does this have any effect on the deb produced?
As long as produced deb ( aspnet package ) doesn't exhibit direct system package dependency, it shouldn't matter. All the distro specific system package is taken care by runtime package.